### PR TITLE
Require fapolicyd rules packages if fapolicyd present

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/foreman/foreman-proxy/foreman-proxy.spec [hunk 1]
```diff
@@ -28,6 +28,9 @@ Requires:       ruby(release) >= 2.5
 Requires:       rubygems
 Requires:       rubygem(bundler_ext)
 
+# Require fapolicyd package if fapoolicyd is present
+Requires: (%{name}-fapolicyd if fapolicyd)
```
Comment (evgeni) [inline packages/foreman/foreman-proxy/foreman-proxy.spec:32]: when this is in place and we've sufficiently tested the flow, I think we can drop the `foreman-fapolicyd → foreman-proxy-fapolicyd` require that you added, as the installation would now be "managed" by the packages here

File: packages/foreman/foreman-proxy/foreman-proxy.spec [hunk 2]
```diff
@@ -28,6 +28,9 @@ Requires:       ruby(release) >= 2.5
 Requires:       rubygems
 Requires:       rubygem(bundler_ext)
 
+# Require fapolicyd package if fapoolicyd is present
```
Comment (evgeni) [inline packages/foreman/foreman-proxy/foreman-proxy.spec:31]:
```suggestion
# Require fapolicyd package if fapolicyd is present
```

fa :hankey: licyd?

Comment (ehelms) [reply packages/foreman/foreman-proxy/foreman-proxy.spec:31]: You said it, not me.

**Comment by ehelms** [comment]
[test rpm]

**Comment by ehelms** [comment]
[test rpm-copr]

**Review by evgeni — APPROVED** [review]
ack pending typo fix (twice)

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Tests are still running. Merge when green.

File: packages/foreman/foreman-proxy/foreman-proxy.spec (hunk 1 above)
Comment (evgeni) [reply packages/foreman/foreman-proxy/foreman-proxy.spec:32]: https://github.com/theforeman/foreman-packaging/pull/9901

**Pull request merged**